### ISSUE TITLE
Options to include (or remove?) public archive

### ISSUE THREAD

**Issue by fraxken** [UlisesGascon/openssf-scorecard-monitor#42]
Hello @UlisesGascon 

The report include archived repository, I think it should not (what do you think?)

For example documentation-ui is archived in the list here:
https://github.com/NodeSecure/Governance/blob/main/reports/ossf_scorecard/report.md

Maybe we need an option to include or exclude public archive

**Comment by UlisesGascon** [comment]
Yes! Great catch @fraxken , we should ignore them in the collection step

**Comment by KoolTheba** [comment]
I'll take the leadership in the research !
